test/utility/test_util.h

```diff
@@ -49,14 +49,21 @@
 #include "windows_shim.h"
 #endif
 
-#define TESTUTIL_ENV_CONFIG_TIERED \
-    ",tiered_storage=(bucket=./"   \
-    "bucket,bucket_prefix=pfx-,local_retention=%d,name=%s)"
+#define DIR_STORE_BUCKET_NAME "bucket"
+#define S3_DEFAULT_BUCKET_NAME "s3testext;ap-southeast-2"
```

Comment:
I'm still bothered by running this from an instance in us east and silently getting results that large because it is accessing halfway around the world.  Of course having a us-east instance would be the same problem.  I would propose that at line 441, when we actually have the bucket name that we'll use, that we call a new function `check_aws_region` with the bucket name.  That function would use [this technique](https://stackoverflow.com/questions/4249488/find-region-from-within-an-ec2-instance) to determine the region of the caller, compare it to the region of the bucket name and make sure they are the same, give an error if not.  We'd probably want some way to override this (some command line option so perhaps it just prints a warning if they are different).  That way, we'd have some way to do a multi-region test if we really wanted to.

If you wanted to put this into a different ticket, I wouldn't object.

Reply:
[WT-10663](https://jira.mongodb.org/browse/WT-10663) is the ticket number for reference.